### DISCUSSION TITLE
error 404 page

### DISCUSSION THREAD

**Discussion by jacitano89** [alandtse/alexa_media_player#2107] [General]
hello. alexa media player  fails while ligging in.  error 404 page. please fix

**Comment by drschoeps** [comment]
Have you tried enabling 2FA on your Amazon account? I have heard that may work, it did not for me. Good luck!

**Reply by jacitano89** [reply]
yes, but to no avail.  I hope you update the link 😔

**Reply by drschoeps** [reply]
guess we'll have to wait for a patch!

**Reply by IP0rTeR** [reply]
Had the same Issue. Enabling 2FA via mobile code worked for me. Thanks!

**Reply by drschoeps** [reply]
Mobile code meaning it uses an authenticator or just SMS numbers? Did you set it up on a PC or on the phone?

**Reply by IP0rTeR** [reply]
I set it up with PC. I enabled the code via SMS

**Reply by drschoeps** [reply]
hmm, didnt work for me, I'll try again! Thanksl :)

**Comment by Sir-Udo** [comment]
Hello, 
Since the update, the error 404 is gone, but it is not possible to type the capture. after i type the captuer, the same side is comming again and again and again. 
my region is germany amazon.de
